crates/codegen/language/definition/src/model/built_ins.rs

```diff
@@ -1,11 +1,19 @@
 use std::rc::Rc;
 
 use codegen_language_internal_macros::{derive_spanned_type, ParseInputTokens, WriteOutputTokens};
+use semver::Version;
 use serde::{Deserialize, Serialize};
 use strum_macros::EnumDiscriminants;
 
 use crate::model::VersionSpecifier;
 
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[derive_spanned_type(Clone, Debug, ParseInputTokens, WriteOutputTokens)]
+pub struct BuiltInContext {
+    pub name: String,
```

Comment:
nit: for consistency with the rest of the model.

```suggestion
    pub name: Identifier,
```